test/ecto/adapters/sqlite3/connection_test.exs

```diff
@@ -2567,32 +2562,25 @@ defmodule Ecto.Adapters.SQLite3.ConnectionTest do
   test "create index with prefix" do
     create = {:create, index(:posts, [:category_id, :permalink], prefix: :foo)}
 
-    assert execute_ddl(create) == [
-             """
-             CREATE INDEX "posts_category_id_permalink_index" \
-             ON "foo"."posts" ("category_id", "permalink")\
-             """
-           ]
+    assert_raise ArgumentError, "SQLite3 does not support table prefixes", fn ->
+      execute_ddl(create)
+    end
 
     create =
       {:create, index(:posts, ["lower(permalink)"], name: "posts$main", prefix: :foo)}
 
-    assert execute_ddl(create) == [
-             """
-             CREATE INDEX "posts$main" ON "foo"."posts" (lower(permalink))\
-             """
-           ]
+    assert_raise ArgumentError, "SQLite3 does not support table prefixes", fn ->
+      execute_ddl(create)
+    end
   end
 
   test "create index with comment" do
-    create =
-      {:create,
-       index(:posts, [:category_id, :permalink], prefix: :foo, comment: "comment")}
+    create = {:create, index(:posts, [:category_id, :permalink], comment: "comment")}
 
     assert execute_ddl(create) == [
              """
              CREATE INDEX "posts_category_id_permalink_index" \
-             ON "foo"."posts" ("category_id", "permalink")\
+             ON "posts" ("category_id", "permalink")\
```

Comment:
Since the test is about comments, I just removed the prefix option here.

Reply:
That's good